### PR TITLE
[blog] submit Melange 3 blog post

### PR DESCRIPTION
**Pull request by anmonteiro**
used a random online translator, still need to check if this looks fine

edit: looks pretty good

image

### CODE REVIEW

File: blog/posts/announcing-melange-3.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+`Js.Float`, `Js.String`, some functions were changed from pipe-first to
+pipe-last and labeled arguments were added; and incorporating those made others
+obsolete, which we removed. Modules such as `Js.List`, `Js.Null_undefined`,
+`Js.Option`, `Js.Result` and `Js.Cast` are also no longer present in Melange 3.
+Alternatives within `Stdlib` or `Belt` are instead​​ recommended.
+
+## New Features and Enhancements
+
+Melange 3 includes a few interesting new features and enhancements. From syntax
+and preprocessing to interop with JS, runtime and error messages, here are some
```
Comment (jchavarri) [inline blog/posts/announcing-melange-3.md:49]: Should we use `JavaScript` word fully?

```suggestion
and preprocessing to interop with JavaScript, runtime and error messages, here are some
```

File: blog/posts/announcing-melange-3.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+    derived by the underlying record.
+
+### Error messages & Hints
+
+Melange 3 provides more informative error messages originating from both the
+`melange.ppx` and the compiler core​​​​.
+
+In this release, we also introduce a new `unprocessed` alert to detect code that
+has made it to the Melange compiler without having been processed by the Melange
+PPX. Besides hinting users to add `(preprocess (pps melange.ppx))` to their dune
```
Comment (jchavarri) [inline blog/posts/announcing-melange-3.md:71]:
```suggestion
PPX. Besides hinting users to add `(preprocess (pps melange.ppx))` to their `dune`
```

File: blog/posts/announcing-melange-3.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+
+For this release, most modules in the `Js` namespace had their APIs unified,
+deduplicated and refactored. In cases such as `Js.Int`, `Js.Date`, `Js.Re`,
+`Js.Float`, `Js.String`, some functions were changed from pipe-first to
+pipe-last and labeled arguments were added; and incorporating those made others
+obsolete, which we removed. Modules such as `Js.List`, `Js.Null_undefined`,
+`Js.Option`, `Js.Result` and `Js.Cast` are also no longer present in Melange 3.
+Alternatives within `Stdlib` or `Belt` are instead​​ recommended.
+
+## New Features and Enhancements
```
Comment (jchavarri) [inline blog/posts/announcing-melange-3.md:46]: Should we mention that Melange is now releasing versions for 4.14 and 5.1? That is quite a big deal, imo.

**Review by jchavarri — APPROVED** [review]
Looks great! I just applied some reformatting to limit width and set separators differently. Just ftr, the first separator in the document has special meaning because it's what we use to pick the "introduction" paragraph(s) that is shown in the blog main page, where all articles are listed. Here is how it looks now:

image

Feel free to move the separator on line 20 to a different place, and merge when you think it's ready. Thanks!

File: blog/posts/announcing-melange-3.md (hunk 3 above)
Comment (anmonteiro) [reply blog/posts/announcing-melange-3.md:46]: great catch! this is definitely something we should mention

**Review by jchavarri — APPROVED** [review]
:shipit:

**Pull request merged**